Assets/Prefabs/Audio/StageAudioSystem.prefab

```diff
@@ -0,0 +1,170 @@
+%YAML 1.1
```

Comment:
this prefab needs to be added to each chapter scene as we create them. Probably there's no need to automate this.